itests/worker_upgrade_test.go

```diff
@@ -1,170 +0,0 @@
-package itests
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/filecoin-project/lotus/api"
-	"github.com/filecoin-project/lotus/itests/kit"
-	"github.com/filecoin-project/lotus/node/config"
-	sealing "github.com/filecoin-project/lotus/storage/pipeline"
-	"github.com/filecoin-project/lotus/storage/sealer/sealtasks"
-	"github.com/filecoin-project/lotus/storage/sealer/storiface"
-)
-
-func TestWorkerUpgradeAbortCleanup(t *testing.T) {
```

Comment:
I think we need a test for upgrading CC sectors.

Reply:
This will get covered as part of https://github.com/filecoin-project/lotus/pull/12017.